parts/k8s/addons/kubernetesmasteraddons-kube-proxy-daemonset.yaml

```diff
@@ -28,9 +28,6 @@ spec:
         - "--feature-gates=ExperimentalCriticalPodAnnotation=true"
         image: "<kubernetesHyperkubeSpec>"
         name: kube-proxy
-        resources:
-          requests:
-            cpu: 100m
         securityContext:
```

Comment:
acc only

Reply:
side question: shall we add agentpool: system to the nodeSelector?